add  custom annoatation for new model export

billmguo opened the pull request:
Summary: add  custom annoatation for new model export

Differential Revision: D77569950

billmguo commented:
@pytorchbot label "topic: not user facing"

winskuo-quic commented on backends/qualcomm/_passes/lift_constant_scalar_operands.py:
```diff
@@ -124,6 +124,10 @@ def _create_tensor_args(
     ) -> Dict[int, TensorConstant]:
         tensor_args = {}
         for i, arg in enumerate(node.args):
+            if hasattr(node.target, "_schema"):
```
Curious under what situation a node has no schema.

billmguo replied:
I will remove this lines, this issue happen when I apply get_quant_embedding_transform for the model embedding layers and export model with -E "4,32"

winskuo-quic reviewed (approved):
LGTM. Thanks

billmguo committed:
add  custom annoatation for new model export (#12126)

Summary:

add  custom annoatation for new model export

Reviewed By: limintang

Differential Revision: D77569950
